src/components/radio-filter/radio-filter.jsx

```diff
@@ -23,6 +24,13 @@ export default React.createClass({
 
     this.props.onChange(choice);
   },
+  componentWillReceiveProps(nextProps){
+    if (nextProps.value !== this.state.activeFilter){
+      this.setState({
+        activeFilter: nextProps.value
+      });
+    }
+  },
```

Comment:
orrect me if i am wrong, this function will execute only when we change props without changing the state of parent component, right?